Sources/Utility/Process.swift

```diff
@@ -196,18 +222,21 @@ public final class Process: ObjectIdentifierProtocol {
         posix_spawn_file_actions_addopen(&fileActions, 0, devNull, O_RDONLY, 0)
 
         var outputPipe: [Int32] = [0, 0]
+        var stderrPipe: [Int32] = [0, 0]
         if redirectOutput {
-            let rv = libc.pipe(&outputPipe)
-            guard rv == 0 else {
-                throw SystemError.pipe(rv)
-            }
+            // Open the pipes.
+            try open(pipe: &outputPipe)
+            try open(pipe: &stderrPipe)
```

Comment:
It would be nice if the client could control this, because sometimes when you want the output merged if you split like this it is really unreadable because the order isn't preserved. We might not need that ability for now, just something to keep in mind...